javascript/javascript2/week2/main.js

```diff
@@ -0,0 +1,45 @@
+console.log('Script loaded');
+
+const products = getAvailableProducts();
+const productsUl = document.querySelector('section.products ul');
+console.log(productsUl);
```

Comment:
Remember to remove console logs that you only use for debugging from your final submission!